pytorch/training/docker/1.10/py3/cu113/Dockerfile.e3.gpu

```diff
@@ -102,10 +102,13 @@ RUN mkdir /tmp/efa \
   && curl -O https://s3-us-west-2.amazonaws.com/aws-efa-installer/aws-efa-installer-${EFA_VERSION}.tar.gz \
   && tar -xf aws-efa-installer-${EFA_VERSION}.tar.gz \
   && cd aws-efa-installer \
+  && apt-get update \
```

Comment:
what this update is used for? are there repo information needs to be refreshed?

Reply:
we have `apt-get clean` in line 92 for reducing image size. this line is to get repo info again